pkg/services/ngalert/notifier/channels/factory.go

```diff
@@ -19,8 +20,8 @@ type FactoryConfig struct {
 	Template *template.Template
 }
 
-// A specialization of store.ImageStore, to avoid an import loop.
 type ImageStore interface {
+	GetImage(ctx context.Context, token string) (*models.Image, error)
```

Comment:
I added this back since I saw how complex the three methods of GetURL, GetFilepath, GetData, made the code. I plan to remove those three after refactoring the existing notifiers back over to use GetImage.